README.md

```diff
@@ -240,4 +240,12 @@ As of February 2021, index resources that don't support tags aren't applicable t
 May 2020: Microsoft.DocumentDB/databaseAccounts/ipRangeFilter updated from a string property to an array.  Please re-author your custom definitions to support the property as an array.  
 July 2020: The alias Microsoft.Sql/servers/securityAlertPolicies/emailAddresses[] and related policies were deprecated. 
 
+### AKS Policy does not current support Application gateway Ingress controller and Nginx controller currently 
+
+The existing ingress policy (i.e. "Kubernetes clusters should be accessible only over HTTPS" ) currently checks only Google ingress controllers and do not currently support AKS ingress controllers. We are working on updating this in the upcoming months to support both controllers. 
+
+### Policy evaluating containers does not support container exclusions currently.  
+
+Some containers are currently marked as non-compliant without ability to modify, such as Istio init containers. This is because some containers like Istio are loaded as side cars, which thus prevents annotation from being set ahead of time. As a solution for this scenario, we are working on updating policy definitions with an 'excludedContainers' parameter to exclude containers in the constraint template and Azure Policy definition in the upcoming months. 
```

Comment:
@nehakulkarni123  FYI, fix for this should be available in production by end of August.

Reply:
Great, added that